src/js/components/sharedComponents/SVGLine.jsx

```diff
@@ -135,7 +135,7 @@ export default class SVGLine extends Component {
             max,
             adjustmentX
         } = this.props;
-        if (isHorizontal) return graphHeight - scale(max);
```

Comment:
@ebdabbs what is happening here?

Reply:
Was this a warm-fix?

Reply:
Oh my bad it's called out in the description of this pr. All good!! 👌 

https://github.com/fedspendingtransparency/usaspending-website/pull/1682/files